chromium_src/net/url_request/url_request_http_job.cc

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "net/url_request/url_request_http_job.h"
+
+#include "net/base/features.h"
+#include "net/cookies/cookie_monster.h"
+
+namespace {
+
+bool ShouldUseEphemeralStorage(net::URLRequestHttpJob* http_job) {
+  if (!base::FeatureList::IsEnabled(net::features::kBraveEphemeralStorage))
+    return false;
+
+  const net::IsolationInfo& isolation_info =
```

Comment:
missing headers for isolation info, and request https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Reply:
Fixed.